src/config.ts

```diff
@@ -12,36 +12,67 @@ export interface ImportSpec {
 }
 
 export interface ValidationConfig {
-
   readonly package: string;
   readonly version: string;
   readonly class: string;
   readonly installEnv?: { [key: string]: any };
   readonly properties?: { [key: string]: any };
 }
 
+export enum SynthesisFormat {
+  PLAIN = 'plain',
+  HELM = 'helm',
+}
+
+export enum HelmChartApiVersion {
+  V1 = 'v1',
+  V2 = 'v2',
+}
+
+export interface Synth {
+  readonly format?: SynthesisFormat;
+  readonly chartApiVersion?: HelmChartApiVersion;
+  readonly chartVersion?: string;
+}
+
 export interface Config {
   readonly app?: string;
   readonly language?: Language;
   readonly output?: string;
   readonly imports?: string[];
   readonly pluginsDirectory?: string;
   readonly validations?: string | ValidationConfig[];
+  readonly synth?: Synth;
 }
 
 const DEFAULTS: Config = {
   output: 'dist',
   pluginsDirectory: path.join(os.homedir(), '.cdk8s', 'plugins'),
+  synth: {
+    format: SynthesisFormat.PLAIN,
+  },
 };
 
 export function readConfigSync(): Config {
   let config: Config = DEFAULTS;
+
   if (fs.existsSync(CONFIG_FILE)) {
     config = {
       ...config,
       ...yaml.parse(fs.readFileSync(CONFIG_FILE, 'utf-8')),
     };
   }
+
+  if (config.synth?.format === SynthesisFormat.HELM) {
+    config = {
+      ...config,
+      synth: {
+        chartApiVersion: HelmChartApiVersion.V2,
```

Comment:
Looks like this default is only applied if the user specified `format: helm` in the config file, but what if the config file is left empty, and the user executed this command:

```console
/Users/epolon/dev/src/github.com/cdk8s-team/cdk8s-cli/bin/cdk8s synth --format helm --chart-version 0.0.1
```

In this case, the default value for `chartApiVersion` is undefined, and indeed, the Chart.yaml file looks like this:

```yaml
apiVersion: "undefined"
name: "cdk8s-helm"
version: "0.0.1"
description: Generated chart for cdk8s-helm
type: application
```

It also makes it ignore CRDs in this case. 

Make sure you add tests for all these cases...